sandbox/routes/patient/patch-patient.js

```diff
@@ -31,7 +31,7 @@ module.exports = [
             }
 
             // Check If-Match header is correct version
-            if (!requestValidator.validateIfMatchHeaderIsCorrectVersion(request, patients.examplePatientSmith)) {
+            if (!requestValidator.validateIfMatchHeaderIsCorrectVersion(request, patients.retrieve.examplePatientSmith)) {
                 throw Boom.preconditionFailed(
                     "This resource has changed since you last read. Please re-read and try again with the new version number.",
```

Comment:
Could you correct this English.  Think it should be 

> This resources has changed since your last read.

Reply:
Changed